lib/matplotlib/axes/_base.py

```diff
@@ -859,7 +862,7 @@ def _update_transScale(self):
         self.transScale.set(
             mtransforms.blended_transform_factory(
                 self.xaxis.get_transform(), self.yaxis.get_transform()))
-        for line in getattr(self, "lines", []):  # Not set during init.
+        for line in getattr(self, "_children", []):  # Not set during init.
```

Comment:
Are we Ok going through all the children instead of the lines?

Reply:
There is no `lines` attribute any more, only a property, which would always exist. But good point, I should probably add an `isinstance` here.